dockerclient/client.go

```diff
@@ -437,7 +464,7 @@ func (e *ClientExecutor) Commit(b *imagebuilder.Builder) error {
 		return fmt.Errorf("unable to commit build container: %v", err)
 	}
 
-	e.Image = image
```

Comment:
@nalind Is this a breaking change for library users ? Custom code which is relying on `ClientExecutor.Image` would now get a `nil` or incorrect value cause we are moving committed image to `ClientExecutor.Committed` 

Just a small concern although I doubt if anybody is going to be using `ClientExecutor` directly.

Reply:
It would, but since there was no indication of whether or not `ClientExecutor.Image` referred to the base image, or the committed image (the caller had to keep track of whether it had committed an image or not), it wasn't ideal.  The only library consumer I know of is the release-3.11 branch of OpenShift Origin, and it doesn't reference this field.

Reply:
@nalind That makes sense.